hotness/bz.py

```diff
@@ -123,10 +123,10 @@ def follow_up(self, text, bug):
         self.log.info("Followed up on bug: %s" % bug.weburl)
 
     def attach_patch(self, filename, description, bug):
-        self.log.debug("Attaching patch to bug %r" % bug.bug_id)
         if os.path.exists(filename) and os.path.getsize(filename) != 0:
+            self.log.debug("Attaching patch to bug %r" % bug.bug_id)
             self.bugzilla.attachfile(bug.bug_id, filename, description,
-                is_patch=True)
+                    is_patch=True)
```

Comment:
Just a thought.  Since this isn't really a patch (but is instead a log file), should we pass ``is_patch=False``?  I don't know what that flag does on the bugzilla.redhat.com side.